css/index.css

```diff
@@ -0,0 +1,289 @@
+    font-weight: bold;
+    color: #555e6c;
+    font-size: 14px;
+}
+
+.container {
+    margin-left: 40px;
+    margin-right: 40px;
+    margin-bottom: 40px;
+}
+
+ul {
+    background-color: blue;
+    list-style-type: none;
+}
+
+li {
+    float: left;
+    padding: 8px 10px;
+}
+
+.hr-rule {
+    height: 1px;
+    border: 0;
+    border-top: 1px solid #f4f3f6;
+    width: 100%;
+}
+
+.nav { 
+    float: left;
+    color: #ccd1d7;
+    font-weight: bold;
+    font-family: 'Manrope', sans-serif;
+}
+
+.patch { 
+    font-weight: bold;
+    color: #1a2939;
+}
+
+.depo {
+    color: #47a8d1;
+}
+
+.nav-img {
+    float:right;
+    margin-right: 20px;
+}
+
+img {
+    border-radius: 25px;
+    height: 25px;
+    width: 25px;
+}
+
+
+.btn-primary {
+    background-color: #4ecb73;
+    color: #e5fdf9;
+    height: 45px;
+    width: 145px;
+    border-radius: 8px;
+    float: right;
+    margin-left: 868px;
+    font-weight: bold;
+    border: none;
+    margin-bottom: 15px
+}
+
+.float-left {
+    float: left;
+}
+
+.header-text {
+    color: #313e4c;
+    margin-top: 5px;
+    font-weight: normal;
+}
+
+.table {
+    text-align: left;
+}
+
+table {
+    border-collapse: collapse;
+    height: 100%;
+    width: 100%;
+}
+
+th {
+    padding: 10px 0px;
+    background-color: #f9fbfc;
+    font-size: 10px;
+    color: #9aa3b0;
+    letter-spacing: 1px;
+}
+
+td {
+    padding-top: 10px;
+    padding-left: 0px;
+}
+
+.line {
+    border-bottom: 1px solid #f4f3f6;
+}
+
+
+
+th.pr-200 {
+    padding-right: 200px;
+}
+
+th.pl-8 {
+    padding-left: 8px;
+}
+
+th.color-text {
+    color: #1e2b3b;
+}
+   
+  
+td img {
+    float: left;
+    margin-left: 2px;
+    padding: 9px;
+    height: 35px;
+    width: 35px;
+}
+    
+.round-img {
+    height: 25px;
+    width: 25px;
+}
+
+.round-img-over {
+    height: 25px;
+    width: 25px;
+    left: 0px;
+    position: relative;
+}
+
+.round-avtar {
+        height: 25px;
+        width: 25px;
+        left: -25px;
+       position: relative;
+}
+
+span {
+    font-size: 9px;
+    color: #abb1bb;
+}
+
+.name {
+    font-size: 15px;
+    color: #47a8d1;
+}
+
+h3 {
+    color: #8a96a0;
+    margin-bottom: -35px;
+    margin-top: 40px;
+}
+
+  
+.comment {
+    width: 60px;
+    height: 27px;
+    background: #fbe0f5;
+    position: relative;
+    -moz-border-radius: 15px;
+    -webkit-border-radius: 12px;
+    border-radius: 5px;
+    margin-left: 200px;
+    margin-top: -54px;
+    float: left;
+}
+
+.comment:before {
+    content: "";
+    position: absolute;
+    right: 100%;
+    top: 4px;
+    width: 0;
+    height: 0;
+    border-top: 10px solid transparent;
+    border-right: 5px solid #fbe0f5;
+    border-bottom: 10px solid transparent;
+}
+
+.comment p {
+    color: #de89ce;
+    padding-top: 0px;
+    text-align: center;
+    font-size: 12px;
+    margin-top: 5px;
+}
+
+.next {
+    color: black;
+    text-decoration: none;
+    margin-left: 60px;
+    size: 30px;
+}
+
+.second-table {
```

Comment:
@indalgit here again you can have create/use mt-50 (margin-top).

Reply:
okay